include/envoy/http/codec.h

```diff
@@ -384,10 +384,14 @@ struct Http1Settings {
     // Performs proper casing of header keys: the first and all alpha characters following a
     // non-alphanumeric character is capitalized.
     ProperCase,
+    // Performs custom casing of header keys. Rules are defined
+    Custom
```

Comment:
Given that `ProperCase` had a trailing comma, I think style would be to include a trailing coma here.

Also, it looks like the comment is cut off?

Reply:
Done, and fixed comment